### ISSUE TITLE
Icon starts in volume

### ISSUE THREAD

**Issue by ElGatoSaez** [AurousGroup/Bug-Tracker#44]
Volume icon starts in Volume 0 but music plays with 100% volume. 
![image](https://cloud.githubusercontent.com/assets/10396037/10412487/c04a0690-6f5d-11e5-8986-74ec11bc37e7.png)

**Comment by srcnalt** [comment]
You can see the same issue here #2

**Comment by ElGatoSaez** [comment]
Sorry about it

**Comment by Radww** [comment]
Go to "C:\Program Files (x86)\Aurous Group\Aurous\data", move index.html to desktop, edit index.html with notepad++, sublime text or other text editor, then search for "<div id="volumeSetting">" it should be at line 611, go to line 612 and edit old line with this one: "<input id="song-volume-slider" data-slider-id="songVolumeSlider" type="text" data-slider-min="0" data-slider-max="100" data-slider-step="1" data-slider-value="100" data-slider-handle="round"/>" (without "")
Or if you are lazy just download this https://mega.nz/#!eohSVJIC!pCrOdZ82cXnuUwIigsnVA3JC316IuShFU3xE-rycZeI and replace with your old index.html (p.s. make a backup before). Hf.

**Comment by ElGatoSaez** [comment]
Will test it. I need to close Aurous?

**Comment by Radww** [comment]
Yup, you have to restart Aurous. I'm going to eat, I'll fix more things when I'll come back.

**Comment by srcnalt** [comment]
Oh I forgot it was open source :smile: I can plunder the errors now.

**Comment by ElGatoSaez** [comment]
Ready. Line 611

This is how it will go on:
								<input id="song-volume-slider" data-slider-id="songVolumeSlider" type="text" data-slider-min="0" data-slider-max="100" data-slider-step="1" data-slider-value="100" data-slider-handle="round"/>

**Comment by haydster7** [comment]
Wouldn't it be more accurate to put the value at what the actual value is, rather than a set amount that could be the actual value or could be some other value.
Good temp fix though, thank you Radww

**Comment by haydster7** [comment]
For me the time through the song (even though no song is playing) Also seems to always be at 1:04. Ill have a look in the index for that :)

**Comment by haydster7** [comment]
Aha, I think I have the culprit
on line 601
`"data-slider-value="14"`
should be
`data-slider-value="0"`

**Comment by haydster7** [comment]
Hmmm, even though the slider is fixed, the value at the start of the bar still says "1:04"

**Comment by haydster7** [comment]
Culprit found
on line 598
`<div class="pull-left" id="song-current-time">1:04</div>`
should be
`<div class="pull-left" id="song-current-time">0:00</div>`

**Comment by haydster7** [comment]
[Index with all repairs](https://drive.google.com/file/d/0B6ArEL92cWBxb1JBckdyVWctWEU/view?usp=sharing)

**Comment by ianchanning** [comment]
The `data-slider-value="0"` is just a placeholder that should be updated when the application starts based on the value that was set the last time aurous was open. However either the code for that isn't written or there's a bug stopping it from happening. Changing the value to 100 will fix it for now but it might cause problems later if the Aurous code is checking for '0' which means unset.
